documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -1,342 +1,462 @@
 [[building]]
-== Building the kernel
+== Build the kernel
 
-The default compilers and linkers that come with an OS are configured to build executables to run on that OS - they are native tools - but that doesn't have to be the case. A cross-compiler is configured to build code for a target other than the one running the build process, and using it is called cross-compilation.
+The default compilers and linkers distributed with an OS are configured to build executables to run on that OS. **Cross-compilation** is the process of building code for a target other than the one running the build process.
```

Comment:
Might be worth clarifying that the former process is what gets referred to as a "native build" ?